lib/FirebaseUtilities.ts

```diff
@@ -8,49 +8,49 @@ import {
   arrayRemove,
   collection,
   setDoc,
-} from 'firebase/firestore/lite'
-import { Score } from '../components/Valuation/LandLikeBox'
-import { Metaverse } from './enums'
+} from "firebase/firestore/lite";
+import { Score } from "../components/Valuation/LandLikeBox";
+import { Metaverse } from "./enums";
 
 // Firebase Init
 const firebaseConfig = {
   apiKey: process.env.FIREBASE,
-  authDomain: 'mgh-app-d6122.firebaseapp.com',
-  projectId: 'mgh-app-d6122',
-  storageBucket: 'mgh-app-d6122.appspot.com',
-  messagingSenderId: '335800469615',
-  appId: '1:335800469615:web:a90434b4ff8ff6e9c2259a',
-  measurementId: 'G-SYMJ1J305Q',
-}
-const app = initializeApp(firebaseConfig)
-const db = getFirestore(app)
+  authDomain: "mgh-app-d6122.firebaseapp.com",
+  projectId: "mgh-app-d6122",
+  storageBucket: "mgh-app-d6122.appspot.com",
+  messagingSenderId: "335800469615",
```

Comment:
I think you are using Prettier. I use it too : ).  So far I've been using single quotes for Strings but we can decide with the team and then we can use the same prettier config to have more unified code.